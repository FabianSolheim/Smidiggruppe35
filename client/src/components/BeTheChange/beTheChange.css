.container {
    max-height: 1130px;
    max-width: 1440px;
    margin: auto;
    width: 90%;
}
.image1 {
    background-image: url("../../images/bethechangeimg1.png");
    background-repeat: no-repeat;
    background-size: cover;
    height: 453px;
}
.text1 {
    width: 100%;
    background-color: rgb(3, 79, 122, 0.1);
    height: 453px;
}
.image2 {
    background-image: url("../../images/beTheChange2.png");
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
    height: 453px;
}
.text2 {
    width: 100%;
    background-color: rgb(3, 79, 122, 0.1);
    height: 453px;
}
.description {
    font-size: 20px;
}
.description em {
    color: #034F7A
}

