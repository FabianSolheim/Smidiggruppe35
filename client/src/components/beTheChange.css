.image1 {
    width: 100%;
    background-image: url("../images/bethechangeimg1.png");
    background-repeat: no-repeat;
    background-size: auto;
    height: 400px;
}
.text1 {
    width: 100%;
    background-color: green;
    height: 400px;
}
.image2 {
    width: 100%;
    background-color: blue;
    height: 400px;
}
.text2 {
    width: 100%;
    background-color: yellow;
    height: 400px;
}

