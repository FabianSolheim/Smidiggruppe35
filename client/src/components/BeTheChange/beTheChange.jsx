import React from "react";
import Box from "@mui/material/Box";
import Grid from "@mui/material/Grid";
import "./beTheChange.css";
import { HeadlineWithUnderline } from "../HeadlineWithUnderline/headlineWithUnderline";
import { Container } from "@mui/material";

export function BeTheChange() {
  return (
    <>
      {HeadlineWithUnderline(
        "Be the Change",
        200,
        "#034f7a",
        0,
        "#212121",
        "36px"
      )}
      <div>
        <p
          style={{
            textAlign: "center",
            marginTop: 20,
            marginBottom: 20,
          }}
        >
          A few highlights from what our members has helped us accomplish
          <br /> so far.
        </p>
      </div>
      <Box sx={{ flexGrow: 1 }}>
        <Grid container spacing={0} style={{ maxWidth: 1440, margin: "auto" }}>
          <Grid item xs={12} md={6} order={{ xs: 2, md: 1 }}>
            <div className="image1" />
          </Grid>
          <Grid item xs={12} md={6} order={{ xs: 1, md: 2 }}>
            <div className="section1">
              <img
                className="KristianiaImg"
                src={require("../../images/KristianiaUniversityCollege_sort_rgb 1.svg")}
              />
              <Container>
                <p className="text1">
                  The donations{" "}
                  <strong>
                    <em>Kristiania University College</em>
                  </strong>{" "}
                  has made has contributed with us being able to build{" "}
                  <strong>
                    <em>50</em>
                  </strong>{" "}
                  more schools.
                </p>
              </Container>
            </div>
          </Grid>
        </Grid>
      </Box>
      <div style={{ height: 30 }} />
      <Box>
        <Grid container spacing={0} style={{ maxWidth: 1440, margin: "auto" }}>
          <Grid item xs={12} md={6}>
            <div className="section2">
              <img
                className="PhotoFindrImg"
                src={require("../../images/PhotoFindr.svg")}
              />
              <Container>
                <p className="text2">
                  <strong>
                    <em>PhotoFindr</em>
                  </strong>{" "}
                  has helped us build wells and give clean water to{" "}
                  <strong>
                    <em>10</em>
                  </strong>{" "}
                  different villages, that never have had access to it before
                </p>
              </Container>
            </div>
          </Grid>
          <Grid item xs={12} md={6}>
            <div className="image2" />
          </Grid>
        </Grid>
      </Box>
    </>
  );
}
