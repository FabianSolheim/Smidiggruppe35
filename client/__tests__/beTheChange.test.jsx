import React from "react";
import ReactDOM from "react-dom";
import { BeTheChange } from "../src/components/BeTheChange/beTheChange";

describe("Be the change test", () => {
  it("shows be the change component", () => {
    const domElement = document.createElement("div");
    ReactDOM.render(<BeTheChange />, domElement);
    expect(domElement.innerHTML).toMatchSnapshot();
  });
});
