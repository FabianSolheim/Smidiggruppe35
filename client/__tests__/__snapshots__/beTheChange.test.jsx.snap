// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Be the change test shows be the change component 1`] = `"<div><div style=\\"font-size: 36px; text-align: center; color: rgb(33, 33, 33); font-family: Josefin Sans; font-weight: 500;\\">Be the Change</div></div><div style=\\"width: 200px; height: 2px; background-color: rgb(73, 3, 116); margin: 5px auto 0px auto;\\"></div><div><p style=\\"text-align: center; margin-top: 20px; margin-bottom: 20px;\\">A few highlights from what our members has helped us accomplish<br> so far.</p></div><div class=\\"MuiBox-root css-i9gxme\\"><div class=\\"MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root\\" style=\\"max-width: 1440px; margin: auto;\\"><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6 css-11ndx0o-MuiGrid-root\\"><div class=\\"image1\\"></div></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6 css-ssy72w-MuiGrid-root\\"><div class=\\"section1\\"><img class=\\"KristianiaImg\\" src=\\"test-file-stub\\"><div class=\\"MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root\\"><p class=\\"text1\\">The donations <strong><em>Kristiania University College</em></strong> has made has contributed with us being able to build <strong><em>50</em></strong> more schools.</p></div></div></div></div></div><div style=\\"height: 30px;\\"></div><div class=\\"MuiBox-root css-0\\"><div class=\\"MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root\\" style=\\"max-width: 1440px; margin: auto;\\"><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6 css-1fyyp8j-MuiGrid-root\\"><div class=\\"section2\\"><img class=\\"PhotoFindrImg\\" src=\\"test-file-stub\\"><div class=\\"MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root\\"><p class=\\"text2\\"><strong><em>PhotoFindr</em></strong> has helped us build wells and give clean water to <strong><em>10</em></strong> different villages, that never have had access to it before</p></div></div></div><div class=\\"MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6 css-1fyyp8j-MuiGrid-root\\"><div class=\\"image2\\"></div></div></div></div>"`;
